README.rst

```diff
@@ -45,21 +48,32 @@ Getting Started
 
       $ make test.run_elasticsearch
 
-3. Create a user and authentication token. Note that the user will be
+#. Create a user and authentication token. Note that the user will be
    created if one does not exist.
 
    ::
 
        $ ./manage.py set_api_key <username> <token>
 
-4. Run the server:
+#. Run the server:
 
    ::
 
        $ ./manage.py runserver
 
 .. _JDK 1.8: https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
 
+Development with edx-enterprise-data
+------------------------------------
+If you need to make changes to ``edx-enterprise-data`` and have them reflected when you run the ``edx-analytics-data-api`` server,
+you can follow these steps. If you do not intend to make changes to ``edx-enterprise-data``, you can skip this section.
+
+#. Recommended: Install this repo into a subfolder of your working directory. Within that subfolder create an ``src`` folder.
+#. Clone the `edx-enterprise-data <https://github.com/edx/edx-enterprise-data>`_ repo into the ``src`` folder.
+#. ``cd`` into your ``edx-data-analytics-api`` folder and activate your virtualenv.
+#. Run ``/src/edx-enterprise-data``.
```

Comment:
is this `/edx/src/edx-enterprise-data` when in the shell?

Reply:
Not for me. My file structure is:
```
edx
  |
  analytics
    |
    edx-analytics-data-api
    src
        |
        edx-enterprise-data
```

Oh, but that should be a pip install command. Let me update.